interp/module.go

```diff
@@ -36,12 +37,25 @@ type Ctxt struct {
 type ModuleExec func(ctx Ctxt, name string, args []string) error
 
 func DefaultExec(ctx Ctxt, name string, args []string) error {
-	cmd := exec.CommandContext(ctx.Context, name, args...)
+	cmd := exec.CommandContext(context.Background(), name, args...)
 	cmd.Env = ctx.Env
 	cmd.Dir = ctx.Dir
 	cmd.Stdin = ctx.Stdin
 	cmd.Stdout = ctx.Stdout
 	cmd.Stderr = ctx.Stderr
+
+	// send Interrupt on cancel instead of Kill (Go's default) when possible
+	go func() {
+		<-ctx.Context.Done()
+		sig := os.Interrupt
+		// Go on Windows doesn't support send Interrupt:
+		// https://golang.org/pkg/os/#Process.Signal
+		if runtime.GOOS == "windows" {
+			sig = os.Kill
+		}
```

Comment:
Will this ever be fixed, or is it fixable?

Reply:
Seems that no: https://github.com/golang/go/issues/6720